edexOsgi/com.raytheon.uf.edex.plugin.grid/utility/common_static/base/bundles/grid/HeightComparison.xml

```diff
@@ -140,7 +140,7 @@
 								<constraint constraintValue="${level}" constraintType="IN" />
 							</mapping>
 							<mapping key="info.datasetId">
-								<constraint constraintValue="NAM40" constraintType="EQUALS" />
+								<constraint constraintValue="NAM12" constraintType="EQUALS" />
```

Comment:
Does any title or anything else need to also be changed from 40 to 12?

Reply:
No, the menu just says "Comparison Families" so it's not specified anywhere else